### ISSUE TITLE
petitboot should not clear terminal scrollback

### ISSUE THREAD

**Issue by npiggin** [open-power/petitboot#75]
It is a bit annoying to lose terminal scrollback from before the kernel boot.

I think it would be nice to behave like grub -- keep scrollback, clear the screen when booting into the new kernel but leave scrollback and a snapshot of the last petitboot screen in the scrollback history.

**Comment by bdragon28** [comment]
Yes. Additionally, it should not turn the cursor off (this is especially annoying when trying to edit the kernel command line on a boot entry.)

I believe it's possible to instruct curses to leave that stuff on, but I'm not sure what the incantation is.
